tests/ints/src/volatile.c

```diff
@@ -44,4 +44,9 @@ void entry3(const unsigned buffer_size, int buffer[])
     buffer[8] = s.buffer[3];
 }
 
-
+void volatile_stuff(void)
+{
+    volatile int x1 = 0;
+    int x2 = x1++;
+    x2;
+}
```

Comment:
I know you said #1064 doesn't have anything to do with it being a loop, and just being `volatile`, but could we just include the loop version here (in addition) anyways, just erring on the more comprehensive testing side?

Reply:
Done.